setup.py

```diff
@@ -480,7 +480,7 @@ def get_file_list (self):
     },
     # Requirements, usable with setuptools or the new Python packaging module.
     install_requires = [
-        'requests >= 2.2.0',
+        'requests<2.15,>=2.2',
```

Comment:
this is odd - this is an issue with requests? i don't understand how `>=2.2` would be better than `>=2.2.0`, they should be semantically equivalent...